e2e/pages/overridden.tsx

```diff
@@ -6,6 +6,7 @@ export default () => (
   <>
     <NextSeo
       noindex={true}
+      nofollow={true}
```

Comment:
Could you add a quick check to `seo.spec.js > SEO overrides apply correctly` to ensure this being set correctly?

Reply:
I added `nofollow` [to the spec already](https://github.com/nik-john/next-seo/blob/nik-john/add-nofollow/cypress/e2e/seo.spec.js#L122) - do you want me to update it to go `index,nofollow` or `noindex,follow`?

Reply:
😅 Oh yeah, it was already set to `nofollow` my bad.